forms/HeaderField.php

```diff
@@ -8,52 +9,76 @@
  * @subpackage fields-dataless
  */
 class HeaderField extends DatalessField {
-	
 	/**
-	 * @var int $headingLevel The level of the <h1> to <h6> HTML tag. Default: 2
+	 * The level of the <h1> to <h6> HTML tag.
+	 *
+	 * @var int
 	 */
 	protected $headingLevel = 2;
-	
+
+	/**
+	 * @param string $name
+	 * @param null|string $title
+	 * @param int $headingLevel
+	 */
 	public function __construct($name, $title = null, $headingLevel = 2) {
-		// legacy handling for old parameters: $title, $heading, ...
-		// instead of new handling: $name, $title, $heading, ...
+		// legacy handling:
+		// $title, $headingLevel...
 		$args = func_get_args();
+
 		if(!isset($args[1]) || is_numeric($args[1])) {
-			$title = (isset($args[0])) ? $args[0] : null;
```

Comment:
`if it's not null, set it to that value. if it is null, set it to null`. Seems super redundant?